opencti-platform/opencti-front/src/private/components/cases/CaseUtils.tsx

```diff
@@ -0,0 +1,93 @@
+import { graphql, ConnectionHandler, Variables } from 'react-relay';
+
+export const generateConnectionId = ({ recordId, key, params }: { recordId?: string, key: string, params?: Variables }) => ConnectionHandler.getConnectionID(recordId ?? 'root', key, params ?? {});
```

Comment:
No, because I don't think this should be extended right now as it a particular case. If in the future we have a similar behavior we will move it and talk about it in dev team :)